user/imports.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Importing shared build configuration
+
+layout: en
+---
+
+The main source of configuration for your build is the `.travis.yml` file stored in your repository. Now you can import up to five configuration snippets into your `.travis.yml`, so you can update multiple repositories making only one change.
```

Comment:
"Now you can" seems like a time reference that would not age well? :) I'd strike "now".